app/logic/api_utils.R

```diff
@@ -70,7 +70,10 @@ get_app_list <- function(
       req_perform() %>%
       resp_body_string() %>%
       fromJSON() %>%
-      filter(app_mode == app_mode_filter)
+      filter(
+        app_mode %in% app_mode_filter,
+        app_role == "owner"
```

Comment:
It would be better to allow the developer to select the role. Please use `config.yml` for that and add an entry to the documentation.